CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ rs-mysql Cookbook CHANGELOG
 
 This file is used to list changes made in each version of the rs-mysql cookbook.
 
+v2.0.2
+------
```

Comment:
you should add that you are allowing the customization of different mysql versions too.